docs/docs-content/vm-management/vm-management.md

```diff
@@ -10,112 +10,47 @@ sidebar_custom_props:
 tags: ["vmo"]
 ---
 
-Palette Virtual Machine Orchestrator (VMO) provides a unified platform for managing containerized and virtualized
-applications. This solution allows organizations to onboard, deploy, manage, and scale VMs within the same cluster as
-their containerized applications. Palette VM Orchestrator simplifies managing infrastructure, improves resource
-utilization, and removes the cost of having a hypervisor.
+Palette Virtual Machine Orchestrator (VMO) provides a unified platform for deploying, managing, and scaling Virtual
+Machines (VMs) and containerized applications within Kubernetes clusters deployed on bare-metal servers in data centers.
 
-![A drawing of VMs deployed to Palette](/docs_vm-mangement_vmo-diagram.webp)
+Palette VMO simplifies infrastructure management, improves resource utilization, and eliminates hypervisor costs.
 
-## Use Cases
-
-Palette VM Orchestrator is particularly suitable in the following scenarios:
-
-- Organizations that want to remove their virtualization infrastructure due to an aging environment or to reduce costs.
-  By using Palette VM Orchestrator, legacy applications and modern, containerized applications can be deployed on VMs.
-
-- Edge locations with a few VMs deployed and where a hypervisor is no longer desired.
-
-## Prerequisites
-
-Palette Virtual Machine Orchestrator requires the following:
-
-- Palette version 3.3.0 or higher.
-
-- For data centers, production VMs are supported on bare metal Kubernetes clusters deployed on Canonical MAAS. To learn
-  how to configure MAAS and create MAAS clusters in Palette, refer to the
-  [Deploy to MAAS](../clusters/data-center/maas/create-manage-maas-clusters.md) guide.
-
-- For Edge deployment, your Edge cluster profile must have a CSI pack. For more information, refer to
-  [Create VMO Profile](./vm-packs-profiles/create-vmo-profile.md#create-the-profile) in the **Edge** tab.
-
-- VMs with Persistent Volume Claim (PVC) must have a StorageClass that supports `ReadWriteMany` (`RWX`) access mode for
-  seamless live migration to a different node - either when triggered manually or during a Kubernetes upgrades.
-
-  :::warning
+![A drawing of VMs deployed to Palette](/vm-mangement_vmo-diagram.webp)
```

Comment:
Excellent diagram and big improvement on this page! It's definitely giving landing page now. 💯 
Romain did a video on VMO relatively recently which might be useful to link https://youtu.be/N7dQ6jmEkCQ?feature=shared